deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/data/tests/EndpointTest.js

```diff
@@ -0,0 +1,71 @@
+const { expect } = require("@playwright/test");
+const { PathManager } = require("./../lib/PathManager.js");
+const { CookieManager } = require("./../lib/CookieManager.js");
+const { ScreenshotManager } = require("./../lib/ScreenshotManager.js");
+
+class EndpointTest {
+
+    page = null;
+    vuContext = null;
+    pathManager = null;
+    cookieManager = null;
+    screenshotManager = null;
+
+    constructor(page, vuContext) {
+        this.page = page;
+        this.vuContext = vuContext;
+        this.pathManager = new PathManager(page);
+        this.cookieManager = new CookieManager(page, vuContext.vars.username, vuContext.vars.session);
+        this.screenshotManager = new ScreenshotManager(page, vuContext.vars.screenshots);
+    }
+
+    // Login with Stored Cookies
+    async restoreSession() {
+        await this.cookieManager.clearCookies();
+        await this.cookieManager.restoreSession();
+    }
+
+    // Access the Dashboard (Endpoints)
+    async accessEndpoint() {
+        await this.pathManager.goto('endpoint-summary');
+        await this.pathManager.waitfor('endpoint-summary');
+
+        await this.screenshotManager.takeAnScreenshot('test_03_endpoint_summary_is_loaded');
+
+        await expect(this.page.getByText('Endpoints')).toBeVisible();
```

Comment:
I think that these functions should be commented explaining their purpose, as it is not entirely clear.

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5629/commits/f46b2fc7da524084f1ae359a11bc2f0f6a02413c